cmd/credentialUtil.go

```diff
@@ -286,14 +286,15 @@ func getCredentialInfoForLocation(ctx context.Context, location common.Location,
 		}
 	}
 
-	if credInfo.CredentialType == common.ECredentialType.OAuthToken() {
-		uotm := GetUserOAuthTokenManagerInstance()
-
-		if tokenInfo, err := uotm.GetTokenInfo(ctx); err != nil {
-			return credInfo, false, err
-		} else {
-			credInfo.OAuthTokenInfo = *tokenInfo
-		}
+	// Get the OAuth token info anyway. This may be used later in STE.
```

Comment:
This change doesn't smell right. Is it necessary?

If user has specifically told us on the command line NOT to use OAuth, why would we try to look up a token?